content/en/real_user_monitoring/browser/collecting_browser_errors.md

```diff
@@ -160,7 +160,7 @@ Get visibility into cross-origin scripts by following these two steps:
 
 2. Configure the `Access-Control-Allow-Origin` HTTP header.
 
-    The most common value for this header is `Access-Control-Allow-Origin: *`, which allows all origins to fetch the resource. Instead, restrict which origins can access your resources by setting, for example `Access-Control-Allow-Origin: www.example.com`.
+    The most common value for this header is `Access-Control-Allow-Origin: *`, which allows all origins to fetch the resource. Otherwise, a set of origins can be allowed but it requires code on the server side to check the origin of the request. In this case, you need to ensure that your origins are allowed by the server. 
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Instead of "Otherwise", I'd say something like "If you want to be stricter,"

<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏It does not require code on the server side, if the customer knows the domains of their website they can hardcode the header